sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -160,9 +162,11 @@ public String getEntityPath() {
      *
      * @return The {@link Mono} the finishes this operation on service bus resource.
      *
-     * @throws NullPointerException if {@code message} is {@code null} , {@code transactionContext} or
-     * {@code transactionContext.transactionID} is {@code null}.
+     * @throws NullPointerException if {@code message}, {@code transactionContext} or
+     *      {@code transactionContext.transactionId} is {@code null}.
      * @throws IllegalStateException if sender is already disposed.
+     * @throws AmqpException if {@code message} is larger than the maximum allowed size of a single message.
```

Comment:
isn't this true that AmqpException will be mapped to `ServiceBusException`. Same in following few API.

Reply:
Fixed this